helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ResourceAccessor.java

```diff
@@ -84,10 +96,66 @@ public Response getResources(@PathParam("clusterId") String clusterId) {
     return JSONRepresentation(root);
   }
 
+  /**
```

Comment:
Partition health related changes are not part of this PR (allow user to change instance config), can we file different issues?